source/documentation/tdb/tdb-xloader.md

```diff
@@ -0,0 +1,51 @@
+---
+title: TDB xloader
+---
+
+TDB xloader ("x" for external) is a bulkloader for very large datasets. The goal
+is stability and reliability for long running loading, running on modest and
+
+xloader is not a replacement for regular TDB1 and TDB2 loaders.
+
+"tdb1.xloader" was called "tdbloader2" and has some improvements.
+
+It is not as fast as other TDB loaders on dataset where the general loaders work
+on without encountering progressive slowdown.
+
+The xloaders for TDB1 and TDB2 are not identical. The TDB2 is more capable; it
```

Comment:
Awkward statement - "The TDB2 is more capable", maybe "TDB2 has more capabilities"